### ISSUE TITLE
It would be possible to add a Spanish keyboard to Toeterm?

### ISSUE THREAD

**Issue by ferlanero** [direc85/toeterm#2]
As I already mentioned on this issue title, I would like to be able to use a spanish keyboard on this great terminal emulator and the spanish translate to the whole application. It would be possible?

**Comment by direc85** [comment]
Yes, it's entirely possible! The more layouts we have, the better!

You can use [the English layout file](https://github.com/direc85/toeterm/blob/master/data/english.layout) as a base and create your desired layout file from there. It shouldn't be too hard, the file looks more intimidating than it really is.

Translation is more straightforward. Download [the base translation file](https://github.com/direc85/toeterm/blob/master/translations/toeterm.ts), rename it accordingly and open it in Qt Linguist (recommened) or your favourite text editor, set the language headers and translate the strings.
